address.config.js

```diff
@@ -1,7 +1,27 @@
 module.exports = {
   whitelists: {
     'nft.owner': ['0xa98A6eefbAE870b88a9C7A43f4b50066A01c93b6'],
-    'datatokens.address': []
+    'datatokens.address': ['0xa2E0747d1FA4006Eff891461758bC96F9cE864F2',
+                           '0xeDBb45Ba45EB6a56072eF224Ff944b7fBd84aE27',
+                           '0x9793Ea60ce9a9B809066ce3872e89c9442dDd4E6'.
+                           '0x4738dbcdea8e215323495882499aae7943ae4c9e',
+                           '0x5000889c6ca759af6dd899fffb05392c0fe02cec',
+                           '0xdf489b1d3304613433eebac48ee2adb2fc3134d9',
+                           '0xe2e72af9879232da54d51b7f24eae88f6d94baef',
+                           '0x18a294bb4dbd6aed6f8bd59853b43372a5b28f3b',
+                           '0xdd545364d9dc5270992d69dc893dac276c176aac'
+                           '0xb0133a4221f6d319e522b3b87448e27bc30a776b',
+                           '0x4738dbcdea8e215323495882499aae7943ae4c9e',
+                           '0xb8ca9fc04082ee46da1be29c286257f6d68a6f85',
+                           '0xc7d8837c835402c17155f90a05f239e2f30a0406',
+                           '0xb56ed2b2e4dbcc40fae70c713758c4b37ace5681',
+                           '0xd5cdc037b198c2d32cfdb91c007f5ff65b2c2ff9',
+                           '0xdd22288a08afd9b1f0d0d6188714cfc7575ce49c',
+                           '0xdd22288a08afd9b1f0d0d6188714cfc7575ce49c',
+                           '0x1f3f1ce4879e0d2319cd553eadcc7b1ef775177a'.
+                           '0x59f93a71b3e3d9c3e94280bafcce49eb02191fdc',
+                           '0x6df7790e95e3bf4406dfc83f6856bc7869861a25',
+                           '0xeab171bfd3ed54b29e97d6dfc92692f7c84da06f']
```

Comment:
```suggestion
    'datatokens.address': [
      '0xa2E0747d1FA4006Eff891461758bC96F9cE864F2',
      '0xeDBb45Ba45EB6a56072eF224Ff944b7fBd84aE27',
      '0x9793Ea60ce9a9B809066ce3872e89c9442dDd4E6',
      '0x5000889c6ca759af6dd899fffb05392c0fe02cec',
      '0xdf489b1d3304613433eebac48ee2adb2fc3134d9',
      '0xe2e72af9879232da54d51b7f24eae88f6d94baef',
      '0x18a294bb4dbd6aed6f8bd59853b43372a5b28f3b',
      '0xdd545364d9dc5270992d69dc893dac276c176aac',
      '0xb0133a4221f6d319e522b3b87448e27bc30a776b',
      '0x4738dbcdea8e215323495882499aae7943ae4c9e',
      '0xb8ca9fc04082ee46da1be29c286257f6d68a6f85',
      '0xc7d8837c835402c17155f90a05f239e2f30a0406',
      '0xb56ed2b2e4dbcc40fae70c713758c4b37ace5681',
      '0xd5cdc037b198c2d32cfdb91c007f5ff65b2c2ff9',
      '0xdd22288a08afd9b1f0d0d6188714cfc7575ce49c',
      '0x1f3f1ce4879e0d2319cd553eadcc7b1ef775177a',
      '0x59f93a71b3e3d9c3e94280bafcce49eb02191fdc',
      '0x6df7790e95e3bf4406dfc83f6856bc7869861a25',
      '0xeab171bfd3ed54b29e97d6dfc92692f7c84da06f'
    ]
```

Reply:
@kaimeinke I removed a couple of duplicates and fixed the separators.

Reply:
Thank you, this one has been messy.